melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/person/PersonModell.java

```diff
@@ -1,17 +1,21 @@
 package no.nav.melosys.eessi.models.person;
 
 import java.time.LocalDate;
+import java.util.Collection;
 
+import lombok.Builder;
 import lombok.Value;
 
 @Value
+@Builder
 public class PersonModell {
     String ident;
     String fornavn;
     String etternavn;
     LocalDate fødselsdato;
-    String statsborgerskapLandkodeISO2;
+    Collection<String> statsborgerskapLandkodeISO2;
     //TPS: status UTPE (utgått) eller UTAN (utgått annullert tilgang)
-    //PDL: status "opphoert" eller "ikkeBosatt" (sistnevnte svarer også til UREG i TPS)
+    //PDL: status "opphoert" svarer til UTPE i TPS
+    // "ikkeBosatt" svarer til UREG i TPS men har også flere betydninger og vil ikke brukes her
```

Comment:
Hvorfor nevner du `ikkeBosatt` hvis det ikke brukes i modellen? Jeg ville kommentere med en blokk (dvs.  /* */) det som  gjelder `erOpphørt` for å vise at det henger sammen.

Reply:
Kun for å dokumentere et sentralt sted at det er overlapp mellom kodene, og at verdien her vil ha en forskjellig betydning avhengig om søket går mot TPS eller PDL.

Reply:
Du svarer ikke helt på spørsmålet. 

Kodene mappes ellers ikke i selve modellen, så jeg synes kommentarene bør være der mappingen skjer og ikke blandes i melosys-eessi sin modell (hvis vi skal ha kommentarer).

Reply:
Jeg synes at det er mest logisk å ha kommentaren her, da det er mest sentralt og man slipper å lese de forskjellige implementasjonene av interfacet for å forstå at feltet kan ha forskjellige betydninger.

Det er jo tross alt for å sammenligne kildene. Det synes jeg blir litt vanskelig å gjøre om man har den dokumentasjonen to steder.

Alt i alt er det nok i praksis bare en huskelapp for meg selv til vi skal sammenligne resultatene mellom PDL og TPS.

Reply:
Oppsummering av tilbakemeldingen min her: enten fjernes det fordi det ikke har noe i modellklassen å gjøre, eller bør man rydde opp og legge til en TODO eller bemerkning.

Reply:
Skal absolutt fjernes på sikt! Ønsker kun å ha det dokumentert sentralt når pdl kjøres i skyggen hvis det skulle komme problemstillinger f.eks. rundt hvorfor vi får fler/færre treff med PDL